### PR TITLE
[IMP] accounting: taxes

### PR DESCRIPTION
**Pull request by toaa-odoo**
Updated the tax documentation for 16.4 onwards. Forward to master.

taskid-3621461

### CODE REVIEW

**Review by LoredanaLrpz — APPROVED** [review]
Hey @toaa-odoo, I'm approving it since I'm off for two weeks, but there's something about the "taxes/labels" anchor to fix (and probably check if this was referenced on the page 
Also, I'm still unsure if this is the best place for the "important admonition", I guess it's fine as it is but lmk if you found a better solution :)

**Review by xpl-odoo — APPROVED** [review]
Pre-approved, just some minor suggestions ;)
@robodoo delegate+

File: content/applications/finance/accounting/taxes.rst [hunk 1]
```diff
@@ -101,181 +86,212 @@ Tax Computation
...
+   .. image:: taxes/invoice-tax-group.png
+      :alt: The Tax Group name is different from the Label on Invoices
 
-In the example above, we see a 0% tax for Intra-Community customers in Europe. It records amounts on
-specific accounts and with specific tax grids. Still, to the customer, it is a 0% tax. That's why
-the :ref:`Label on the Invoice <taxes/label-invoices>` indicates *0% EU*, and the Tax Group name,
-above the *Total* line, indicates *0%*.
+   In the example above, the :guilabel:`0% EU S` tax for intra-community customers in Europe records
+   the amount on specific accounts and tax grids. However, it remains a 0% tax to the customer. This
+   is why the indicates :guilabel:`0% EU S`, and the tax group name above the :guilabel:`Total` line
```
[the remark this answers is not in the document]
Comment (toaa-odoo) [reply content/applications/finance/accounting/taxes.rst:231]: label*

**Comment by toaa-odoo** [comment]
@robodoo r+

**Comment by toaa-odoo** [comment]
@robodoo r+

**Pull request closed without merging**